app/src/main/java/eu/kanade/domain/track/interactor/AddTracks.kt

```diff
@@ -79,19 +79,19 @@ class AddTracks(
 
     suspend fun bindEnhancedTrackers(manga: Manga, source: Source) = withNonCancellableContext {
         withIOContext {
-            getTracks.await(manga.id)
+            trackerManager.loggedInTrackers()
                 .filterIsInstance<EnhancedTracker>()
                 .filter { it.accept(source) }
                 .forEach { service ->
                     try {
                         service.match(manga)?.let { track ->
                             track.manga_id = manga.id
                             (service as Tracker).bind(track)
-                            insertTrack.await(track.toDomainTrack()!!)
+                            insertTrack.await(track.toDomainTrack(idRequired = false)!!)
```

Comment:
This is now needed since the track might not have an id yet.

Reply:
the old implementation didn't need this (which actually did auto bind) either so should be fine? unless it was a issue during testing

Reply:
It was indeed an issue during my tests. If the item you add to the library has no exisiting track with the tracker, the track has no id yet and `toDomainTrack` throws an exception.
It seems like this method allows for inserting and updating, but when you are inserting you have to set this parameter to false.